app/javascript/components/subject/SymptomOnset.js

```diff
@@ -14,6 +15,9 @@ class SymptomOnset extends React.Component {
     this.handleSubmit = this.handleSubmit.bind(this);
     this.submit = this.submit.bind(this);
     this.handleChange = this.handleChange.bind(this);
+    this.systemOnsetToolTip = `Used by the system to determine if the non-test based recovery definition
```

Comment:
symptomOnset, not systemOnset 😄

Reply:
Some of the messages are this.blah = '', and some are functions. Can we do one or the other? I think it's cool the component supports both, but we need to be consistent for maintainability. Probably want to stick with the object version, since it's the most flexible.